collision-handlers.rb

```diff
@@ -0,0 +1,58 @@
+require 'gosu'
+require 'chipmunk'
+require_relative 'player'
+require_relative 'obj'
+
+class PlayerPlatformCollisionHandler
+  def initialize(player)
+    @player = player
+  end
+
+  def begin(a, b, arbiter)
+
+    if arbiter.normal(0).y < 0
+      arbiter.ignore
+      false
```

Comment:
If it returns false then the collision stops if the function returns true the collision continues